### PR TITLE
Cleanup deploy.

### PR DESCRIPTION
**Pull request by vmarmol**
Move Docker image build to deploy/docker
Move configs to deploy/config

Clearing the way for multiple configs.

### CODE REVIEW

**Comment by vishh** [comment]
@vmarmol: The integration test under `integration/kube_test.go` needs to be updated with the new docker file location.

File: docs/influxdb.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ _Warning: Virtual machines need to have at least 2 cores for InfluxDB to perform
 
 ### Start all of the pods and services
 ```shell
-$ kubectl.sh create -f deploy/
+$ kubectl.sh create -f deploy/config/
```
Comment (vishh) [inline docs/influxdb.md]: Should we rename it to `kube-config` instead?

**Comment by vishh** [comment]
```
Error:		No error is expected but got failed to build docker binary ("exit status 1") - "time=\"2015-03-19T17:12:55Z\" level=\"fatal\" msg=\"no Dockerfile found in .\" \n"
```

**Comment by vishh** [comment]
Thanks for the cleanup :) 
LGTM. Will merge once the integration test passes.

File: docs/influxdb.md (hunk 1 above)
Comment (vmarmol) [inline docs/influxdb.md]: Done.

**Comment by vmarmol** [comment]
Ah, I figured I'd miss something :) should be fixed. Will let the integration test decide. Thanks for the quick review @vishh!

**Comment by vmarmol** [comment]
Finally Travis no longer hates me :)

**Comment by vishh** [comment]
Lol. Thanks for the PR!

**Pull request merged**